lib/Room.php

```diff
@@ -238,6 +238,22 @@ public function getToken(): string {
 	}
 
 	public function getName(): string {
+		if ($this->type === self::ONE_TO_ONE_CALL) {
+			if ($this->name === '') {
+				// Fill the room name with the participants for 1-to-1 conversations
+				$users = $this->getParticipantUserIds();
+				sort($users);
+				$this->setName(json_encode($users), '');
```

Comment:
Maybe add a comment to explain that `''` is needed to avoid showing a system message about the name change? Or in `setName` documentation.

Well, I think that is the reason to use `''` as the old name, but if it is a different one... then mention the different one :-P

Reply:
the `''` is needed to avoid issues with nesting as it would call `getName()` (the current function) to get the previous name. Don't care about a system message, it was a nice sideeffect :)